transaction_receipt.go

```diff
@@ -196,10 +215,10 @@ func (receipt TransactionReceipt) _ToProtobuf() *services.TransactionGetReceiptR
 		NodeId:                  receipt.NodeID,
 	}
 
-	if receipt.ExchangeRate != nil {
+	if receipt.ExchangeRate != nil && receipt.NextExchangeRate != nil {
 		receiptFinal.ExchangeRate = &services.ExchangeRateSet{
 			CurrentRate: receipt.ExchangeRate._ToProtobuf(),
-			NextRate:    receipt.ExchangeRate._ToProtobuf(),
+			NextRate:    receipt.NextExchangeRate._ToProtobuf(),
 		}
 	}
```

Comment:
Is it required to have `receipt.NextExchangeRate != nil` on every valid `receipt.ExchangeRate`?

Reply:
Fixed.